Translations/translation_EN.json

```diff
@@ -159,49 +159,49 @@
 				"Boost",
 				"temp"
 			],
-			"desc": "Tip temperature used in \"boost mode\""
+			"desc": "Tip temperature used in \"Boost mode\""
 		},
 		"AutoStart": {
 			"text2": [
 				"Start-up",
 				"behavior"
 			],
-			"desc": "O=off | S=heat up to soldering temp | Z=standby at sleep temp until moved | R=standby without heating until moved"
+			"desc": "O=off | S=heat up to soldering temp | Z=standby at sleep temp until moved | R=standby at no heat until moved"
 		},
 		"CooldownBlink": {
 			"text2": [
 				"Cooldown",
 				"flashing"
 			],
-			"desc": "Flash the temperature reading after heating was halted while the tip is still hot"
+			"desc": "Flash the temperature reading, if tip is still hot, after halting heat"
 		},
 		"TemperatureCalibration": {
 			"text2": [
 				"Calibrate",
-				"temperature"
+				"tip CJC"
 			],
-			"desc": "Calibrate tip Cold Junction Compensation (not required if Delta T is < 5°C)"
+			"desc": "Tip-temperature Cold Junction Compensation; not needed if Tip is +/- 5°C of display"
 		},
 		"SettingsReset": {
 			"text2": [
-				"Restore factory",
+				"Restore default",
 				"settings"
 			],
-			"desc": "Reset all settings to default"
+			"desc": "Reset all default settings for this firmware version"

```

Comment:
How does this make the sense any cleared.
Do you reset the *default settings*, or do you reset the *current settings*?
You see, you correction does take away the simplicity and clearness.

Do you speak another language?
If so, please translate both phrases and judge for your self wich is clearest.